### ISSUE TITLE
retrieve exit code for "event:die"

### ISSUE THREAD

**Issue by cyrilchampier** [logmatic/logmatic-docker#13]
In an ENTRYPOINT shell script, we check for env vars.
If one is missing, we exit with an echo and a code:
```
  if [ -z "$variable_contents" ]; then
    echo >&2 "error: missing \"$1\" environment variable"
    exit 1
  fi
```

(The echo does not appear in logmatic, even if it appears on docker logs, but already signaled, not the point of this ticket)

Would it be possible to send the exit code to logmatic with "event:die" ?

**Comment by gpolaert** [comment]
@cyrilchampier 
I've added the exit_code when it's available. If you want to test it, you can use the logmatic/logmatic-docker:dev image.

![image](https://cloud.githubusercontent.com/assets/1678246/22366754/a573e6dc-e47e-11e6-8bb5-38c877faccad.png)

**Comment by cyrilchampier** [comment]
yep, working, thanks !

**Comment by gpolaert** [comment]
Merged to 1.2.1
